Cargo.toml

```
@@ -9,8 +9,8 @@ crate-type = ["cdylib", "rlib"]
 
 [dependencies]
 serde = { version = "1.0", features = ["derive"] }
-serde_json = { git = "https://github.com/nearprotocol/json", rev = "1f5779f3b0bd3d2a4b0b975abc46f3d3fe873331", features = ["no_floats"] }
-near-bindgen = { version = "0.6.0" }
+serde_json = "1.0.52"
+near-sdk = "0.9.2"
 borsh = "0.6.1"
 wee_alloc = "0.4.5"
```

```suggestion
serde = { version = "*", features = ["derive"] }
serde_json = "*"
borsh = "*"
near-sdk = "0.9.2"
wee_alloc = { version = "0.4.5", default-features = false, features = [] }
```
This way you can make sure there are no different versions of common libraries.

---

Thanks!